modules/rostests/apitests/appshim/versionlie.c

```diff
@@ -62,7 +62,7 @@ static void verify_shima_imp(PHOOKAPI hook, const VersionLieInfo* info, PCSTR sh
 
     while (v1.dwOSVersionInfoSize)
     {
-        ok1 = GetVersionExA((LPOSVERSIONINFOA)&v1), ok2;
```

Comment:
This warning sounds like a bug in clang. There is no reason to issue an "expression result unused" warning here. There is however, an uninitialized variable that's being used.
That said, the fix looks correct.